test/system/curiosity_cards_test.rb

```diff
@@ -0,0 +1,137 @@
+# frozen_string_literal: true
+
+require 'application_system_test_case'
+# rubocop:disable Metrics/ClassLength
```

Comment:
This can be done on `rubocop.ym` file instead. We don't care about the ClassLength on all spec files so
You don't have to change this but keep it in mind